pom.xml

```diff
@@ -0,0 +1,86 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>uk.gov.companieshouse</groupId>
+    <artifactId>companies-house-parent</artifactId>
+    <version>1.3.1</version> <!-- lookup parent from repository -->
+  </parent>
+  <artifactId>auth-code-notification</artifactId>
+  <version>unversioned</version>
+  <name>auth-code-notification</name>
+  <properties>
+    <java.version>11</java.version>
+    <spring-boot-dependencies.version>2.6.1</spring-boot-dependencies.version>
+    <spring-boot-maven-plugin.version>2.6.1</spring-boot-maven-plugin.version>
+    <log4j.version>2.17.2</log4j.version>
+    <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
+    <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
+    <org.mapstruct.version>1.5.0.RC1</org.mapstruct.version>
```

Comment:
not sure we'll need mapstruct, we can add it at a later date if we need it so i'd take that dependency out.

Reply:
Removed